src/librbd/cache/pwl/rwl/replica/main.cc

```diff
@@ -0,0 +1,181 @@
+#include <inttypes.h>
```

Comment:
like immutable_object_cache,  it's better in src/tools directory.

Reply:
done